### PR TITLE
Rename estimagic to optimagic

### PR DESCRIPTION
**Pull request by timmens**

### CODE REVIEW

File: index.md [hunk 1]
```diff
@@ -14,7 +14,7 @@ fashion?
 We thus develop software of various kinds. On this site we group it into
 
 1. [**General Purpose Tools**](/tools/). Our most developed package is
-   [estimagic](http://github.com/opensourceeconomics/estimagic), a library for
+   [optimagic](http://github.com/opensourceeconomics/optimagic), a library for
```
Comment (hmgaudecker) [inline index.md:17]: link

File: tools.md [hunk 2]
```diff
@@ -3,20 +3,19 @@ layout: page
...
-including a unified interface that supports a large number of local and global
-optimization algorithms and the possibility of monitoring the optimization procedure via
-a beautiful interactive dashboard. It provides tools for nonlinear optimization,
-numerical differentiation, and statistical inference.
-[(Documentation)](https://estimagic.readthedocs.io){:target="_blank"}
+### optimagic
+optimagic is a Python package for numerical optimization. It is a unified interface to
+optimizers from SciPy, NlOpt and many other Python packages. optimagic’s `minimize`
+function works just like SciPy’s, but you get more optimizers for free. On top you get
+powerful diagnostic tools, parallel numerical derivatives and more. optimagic was
```
Comment (hmgaudecker) [inline tools.md:10]: Something like: "We started developing optimagic under the name estimagic out of a need to robustify estimation of nonlinear economic models, including statistical inference. That functionality is now a subpackage of optimagic, which is still called estimagic. Optimagic now has its own organization on Github: [optimatic-dev](link)."

?

File: tools.md [hunk 3]
```diff
@@ -3,20 +3,19 @@ layout: page
...
+formerly called estimagic, because it also provides functionality to perform statistical
+inference on estimated parameters. estimagic is now a subpackage of optimagic.
+[(Documentation)](https://optimagic.readthedocs.io){:target="_blank"}
 
 ### pybaum
 pybaum provides tools to work with pytrees which is a concept borrowed from JAX.
 Pytree's refer to tree-like structures built out of container-like Python objects. We
-use pybaum in several of our packages. One notable example is estimagic.
+use pybaum in several of our packages. One notable example is optimagic.
```
Comment (hmgaudecker) [inline tools.md:18]: Still true? Future-proof?

Comment (timmens) [reply tools.md:18]: Currently, still true, but it will probably be replaced at some point.

File: tools.md (hunk 2 above)
Comment (timmens) [reply tools.md:10]: I like it. Do you imagine this text as a replacement for the current text or as a preamble?

File: tools.md (hunk 3 above)
Comment (hmgaudecker) [reply tools.md:22]: Then just remove that sentence?

File: tools.md (hunk 2 above)
Comment (hmgaudecker) [reply tools.md]: Sorry, formatting was not so nice. I would actually have left the original, replacing "optimagic was formerly ... and statistical inference." with what I wrote.

File: tools.md [hunk 4]
```diff
@@ -3,20 +3,20 @@ layout: page
...
-inferences about the estimated model parameters. It is especially suited to solve
-difficult optimization problems. It provides several advantages over similar packages,
-including a unified interface that supports a large number of local and global
-optimization algorithms and the possibility of monitoring the optimization procedure via
-a beautiful interactive dashboard. It provides tools for nonlinear optimization,
-numerical differentiation, and statistical inference.
-[(Documentation)](https://estimagic.readthedocs.io){:target="_blank"}
+### optimagic
+optimagic is a Python package for numerical optimization. It is a unified interface to
+optimizers from SciPy, NlOpt and many other Python packages. optimagic’s `minimize`
```
Comment (hmgaudecker) [inline tools.md:8]:
```suggestion
optimizers from SciPy, NlOpt, and many other Python packages. optimagic’s `minimize`
```
Oxford comma fanboy :shrug:

File: tools.md [hunk 5]
```diff
@@ -3,20 +3,20 @@ layout: page
...
+function works just like SciPy’s, but you get more optimizers for free. On top you get
+powerful diagnostic tools, parallel numerical derivatives and more. We started
```
Comment (hmgaudecker) [inline tools.md:10]:
```suggestion
powerful diagnostic tools, parallel numerical derivatives, and more. We started
```

File: tools.md [hunk 6]
```diff
@@ -3,20 +3,20 @@ layout: page
...
+nonlinear economic models, including statistical inference. That functionality is now a
+subpackage of optimagic, which is still called estimagic. Optimagic now has its own
+organization on Github: [optimagic-dev](https://github.com/optimagic-dev).
+[(Documentation)](https://optimagic.readthedocs.io){:target="_blank"}
 
 ### pybaum
 pybaum provides tools to work with pytrees which is a concept borrowed from JAX.
-Pytree's refer to tree-like structures built out of container-like Python objects. We
-use pybaum in several of our packages. One notable example is estimagic.
+Pytree's refer to tree-like structures built out of container-like Python objects.
```
Comment (hmgaudecker) [inline tools.md:19]:
```suggestion
Pytrees refer to tree-like structures built out of container-like Python objects.
```

**Review by hmgaudecker — APPROVED** [review]
Thanks!

**Pull request merged**